tokio/tests/time_interval.rs

```diff
@@ -209,3 +209,76 @@ fn poll_next(interval: &mut task::Spawn<time::Interval>) -> Poll<Instant> {
 fn ms(n: u64) -> Duration {
     Duration::from_millis(n)
 }
+
+mod tmp_tests {
```

Comment:
Why the module?

Reply:
Initially I intended to keep the specific helper struct with the `Stream` implementation and the test together, but I guess it's not needed. I removed it.